### DISCUSSION TITLE
禁用首页的悬浮视频, 为单独的一个插件选项

### DISCUSSION THREAD

**Discussion by HornyPrivateGamer** [the1812/Bilibili-Evolved#5053] [功能建议 / Ideas]
目前想要禁止首页的悬浮视频的话，只有开启 清爽首页 或者 极简首页.

但是有些人喜欢原版的排版. 但是又非常讨厌悬浮视频. 希望可以单独开启，谢谢大佬.

**Comment by the1812** [comment]
可以换 BewlyBewly, 排版比较接近原版

**Reply by HornyPrivateGamer** [reply]
BewlyBwely 还是跟原版有差别的. 希望大佬能不能单独出个插件. 悬浮视频真的很讨厌. 谢谢麻烦了

**Comment by HornyPrivateGamer** [comment]
@the1812 
.bili-video-card .bili-video-card__image--hover:hover .bili-video-card__mask {
    visibility: visible !important;
    opacity: 1 !important;
}


www.bilibili.com##.v-inline-player
www.bilibili.com##.v-inline-player.mouse-in.visible

.video-page-card-small .card-box .pic-box .duration {
    visibility: visible !important;
    opacity: 1 !important;
    display: block !important;
}


Using Ad blocker + Stylish

**Comment by CloudyEagle** [comment]
请问您这个悬浮视频是通过稍后再看弹出的悬浮视频吗？
